bundle/config/testdata/duplicate_resource_name_in_subconfiguration_job_and_job/databricks.yml

```diff
@@ -0,0 +1,10 @@
+bundle:
+  name: test
+
+workspace:
+  profile: test
+
+resources:
+  jobs:
+    foo:
+      name: job foo
```

Comment:
If the test focuses on just the duplicate jobs and hardcodes the loads, these can be `resources1` and `resources2`, for example. I was looking at this and thinking this won't work because there is no `include` section in this file, while it does because the loads are explicit.